core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLPlainParamSuiteSaxonEE.scala

```diff
@@ -0,0 +1,124 @@
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.rackspace.com.papi.components.checker
+
+import com.rackspace.cloud.api.wadl.Converters._
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class ValidatorWADLPlainParamSuiteSaxonEE extends BaseValidatorSuite {
+  val badXML_Plain1 = <e xmlns="http://www.rackspace.com/repose/wadl/checker/step/test">
+                        <id>21f1fcf6-bf38-11e1-878e-133ab65fcec3</id>
+                        <even>22</even>
+                     </e>
+  val badXML_Plain2 = <a xmlns="http://www.rackspace.com/repose/wadl/checker/step/test"
+                        id="21f1fcf6-bf38-11e1-878e-133ab65fcec3"
+                        even="22"/>
+
+  //
+  // Like validator_XSDElementContentPlainOptMsgCode but using Saxon
+  //
+  val validator_XSDElementContentPlainOptMsgCodeS = Validator((localWADLURI,
+      <application xmlns="http://wadl.dev.java.net/2009/02"
+                   xmlns:tst="http://www.rackspace.com/repose/wadl/checker/step/test"
+                   xmlns:rax="http://docs.rackspace.com/api">
+        <grammars>
+           <include href="src/test/resources/xsd/test-urlxsd.xsd"/>
+        </grammars>
+        <resources base="https://test.api.openstack.com">
+           <resource path="/a/b">
+               <method name="PUT">
+                  <request>
+                      <representation mediaType="application/xml" element="tst:a">
+                          <param style="plain" path="tst:a/@stepType" required="true" rax:message="No stepType attribute on a" rax:code="500"/>
+                      </representation>
+                      <representation mediaType="application/json"/>
+                  </request>
+               </method>
+               <method name="POST">
+                  <request>
+                      <representation mediaType="application/xml" element="tst:e">
+                          <param style="plain" path="tst:e/tst:stepType" required="true" rax:message="no stepType on e" rax:code="501"/>
+                      </representation>
+                  </request>
+               </method>
+           </resource>
+           <resource path="/c">
+               <method name="POST">
+                  <request>
+                      <representation mediaType="application/json"/>
+                  </request>
+               </method>
+               <method name="GET"/>
+           </resource>
+        </resources>
+    </application>)
+    , TestConfig(false, false, true, true, true, 10,
+                 true, false , false, "SaxonHE", true,
```

Comment:
Should the `xslEngine` here be set to `SaxonEE` instead of `SaxonHE`? The test name seemed to indicate to me that it should, but I don't have a great idea of what's being tested here or why these tests were moved.

Reply:
Yea, this was an old HE test case that now requires a license. I'll go ahead and change to SaxonEE.  Selecting HE will still work though, if you have a license.